backend/drive/drive.go

```diff
@@ -468,20 +470,32 @@ func NewFs(name, path string) (fs.Fs, error) {
 		return nil, errors.Wrap(err, "couldn't create Drive client")
 	}
 
-	// Read About so we know the root path
+	// set root folder for a team drive or query the user root folder
+	if f.isTeamDrive {
+		f.rootFolderID = f.teamDriveID
+	} else {
+		var rootFolder *drive.File
+		err = f.pacer.Call(func() (bool, error) {
+			rootFolder, err = f.svc.Files.Get("root").Fields("id").Do()
```

Comment:
This is doing an extra transaction on startup compared to the old code, which some users will notice (those who use rclone for lots of small things from the command line).

I wonder if we could set it to "root" then resolve it later somehow.  I know we do comparisons against f.rootFolderID in places however those places are all using an ID read from the directory cache which we can put in as root maybe.

For my own reference I looked up [the docs](https://developers.google.com/drive/v3/web/folder) and it says "You can use the alias root to refer to the root folder anywhere a file ID is provided"

I put a reference to this in #320

Reply:
I din't notice the "root" alias in the docs.
I'm testing it now. It would be nice to skip this call.

Reply:
You are right, "root" can be used as the rootFolderID. I just uploaded the change.